docusaurus/docs/user-docs/content-manager/writing-content.md

```diff
@@ -8,31 +8,31 @@ displayed_sidebar: userDocsSidebar
 
 # Writing content
 
-In Strapi, writing content consists in filling up fields, which are meant to contain specific content (e.g. text, numbers, media etc.). These fields were configured for the collection or single type beforehand, through the [Content-type Builder](/user-docs/content-type-builder).
+In Strapi, writing content consists in filling up fields, which are meant to contain specific content (e.g. text, numbers, media, etc.). These fields were configured for the collection or single type beforehand, through the [Content-type Builder](/user-docs/content-type-builder).
 
 ![Edit view to write content](/img/assets/content-manager/edit-view2.png)
 
 ## Filling up fields
 
 To write or edit content:
 
-1. Access the edit view of your collection or single type.
+1. Access the edit view of your collection or a single type.
```

Comment:
```suggestion
1. Access the edit view of your collection type or single type.
```

Your suggestion would change the meaning of the sentence. Strapi has 2 different content types: collection type and single type. :)